src/js/services/incomingData.js

```diff
@@ -84,27 +84,37 @@ angular.module('copayApp.services').factory('incomingData', function($log, $stat
 
     data = sanitizeUri(data);
 
-    // BIP21
-    if (bitcore.URI.isValid(data)) {
-      var parsed = new bitcore.URI(data);
+    // Bitcoin or Bitcoin Cash URL
+    if ((/^bitcoin[cash]*:/).exec(data)) {
```

Comment:
this means: "match a string that starts with "bitcoin" then may or may not have 0-infinity repetitions of any of the following letters ["c","a","s","h"] then a colon.

Notice what matches and what doesn't:
http://refiddle.com/refiddles/59b3564b75622d096f6d0000

Reply:
This is the correct regex: `/^bitcoin(cash)*:/` parenthesis instead of brackets

http://refiddle.com/refiddles/59b3569075622d0967760000

Reply:
+1

Reply:
Fixed. Thanks for review.